File: client-app/src/app/model/ChatDTO.ts
export interface ChatDTOInterface {
  sender:string;
  reciever:string;

}
export class ChatDTO implements ChatDTOInterface {
  sender:string;
  reciever:string;

  constructor(obj: ChatDTOInterface) {
    this.sender = obj.sender;
    this.reciever = obj.reciever;
  }
}

File: ConnectionService/src/main/java/com/example/connectionsservice/Dto/ChatDTO.java
package com.example.connectionsservice.Dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
//@AllArgsConstructor
//@NoArgsConstructor
public class ChatDTO {
    private String sender;
    private String reciever;
	public String getSender() {
		return sender;
	}
	public void setSender(String sender) {
		this.sender = sender;
	}
	public String getReciever() {
		return reciever;
	}
	public void setReciever(String reciever) {
		this.reciever = reciever;
	}
	public ChatDTO(String sender, String reciever) {
		super();
		this.sender = sender;
		this.reciever = reciever;
	}
	public ChatDTO() {}
    
}
